java/vector/src/main/codegen/templates/UnionVector.java

```diff
@@ -690,4 +690,40 @@ public int hashCode(int index) {
     public <OUT, IN> OUT accept(VectorVisitor<OUT, IN> visitor, IN value) {
       return visitor.visit(this, value);
     }
+
+  @Override
```

Comment:
It seems the 3 toString methods have much duplicated code, can we merge them into one?

Reply:
Right, almost same. I thought about this before, the three class only have a common parent interface ValueVector, and we could not have default toString implementation in interface（methods in Object）. The only way to do this seems to put this method in a utility class, is there already exist one or I am not sure is it worth to create a new one?

Reply:
I added ValueVectorUtility and move logic into this class.

Reply:
Sounds good. Thanks.